docs/releases/v1.3.0.md

```diff
@@ -0,0 +1,32 @@
+# OPA Core Module version v1.3.0
+
+SIGHUP team maintains this module updated and tested. That is the main reason why we worked on this new release.
+With the Kubernetes 1.20 release, it became the perfect time to start testing this module against this Kubernetes
+release. Minor works have been done in the constraint templates.
+
+Continue reading the [Changelog](#changelog) to discover them:
+
+## Changelog
+
+- Update OPA Gatekeeper version from: 3.1.1 to [3.2.2](https://github.com/open-policy-agent/gatekeeper/releases/tag/v3.2.2)
+- Include Gatekeeper Policy Manager, GPM, from our amazing @ralgozino. [0.4.0](https://github.com/sighupio/gatekeeper-policy-manager/releases/tag/v0.4.0).
```

Comment:
Once PR #20 is merged, we should update also here the version :)

Reply:
Done!